source/user_guide.md

```diff
@@ -666,35 +666,61 @@ configured staging directory (`Output/Composer` in the default configuration).
    and produces a binary representation of the application supervisor. It
    performs the generation and build steps in sequence.
 
- - `compose /clean`: Given a placed application graph instance (or multiple),
-   clear any compiled binaries composed for that application.
-
  - `compose /decompose`: Given a placed application graph instance (or
    multiple), clear any compiled binaries and generated source files composed
    for that application.
 
- - `compose /degenerate`: Given a placed application graph instance (or
-   multiple), clear any generated source files composed for that application.
-
  - `compose /generate`: Given a placed application graph instance (or
    multiple), generates translation units (source code) from the loaded XML, to
    be compiled into application binaries (both for normal devices and
    supervisor devices).
 
+ - `compose /degenerate`: Given a placed application graph instance (or
+   multiple), clear any generated source files composed for that application.
+
  - `compose /compile`: Given a generated application graph instance (or
    multiple), compiles instruction and data binaries to be loaded onto the POETS
    Engine, and produces a binary representation of the application supervisor.
 
+ - `compose /clean`: Given a placed application graph instance (or multiple),
+   clear any compiled binaries composed for that application.
+
  - `compose /bypass`: Bypasses most of the compose process provided that the
    compiled binaries for the application already exist, allowing the operator
    to reuse binaries from a previous run or to use binaries compiled elsewhere. 
    The loaded application must be identical in terms of definition and placement 
    for this to work - there are no checks beyond binary existance.
 
+ - `compose /reset`: Deletes the Composer, clearing all data, and creates a new one.
+
+ - `compose /dump`: Dumps diagnostic data regardign the Composer to the microlog.
```

Comment:
"regarding"

Reply:
👍 serves me right for trying to use the editor on Github